src/Widgets/AppEntry.vala

```diff
@@ -71,12 +71,15 @@ public class Notifications.AppEntry : Gtk.ListBoxRow {
             active = true
         };
         unowned var expander_style_context = expander.get_style_context ();
-        expander_style_context.add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
         expander_style_context.add_class ("image-button");
 
-        var clear_btn_entry = new Gtk.Button.from_icon_name ("edit-clear-all-symbolic", Gtk.IconSize.SMALL_TOOLBAR) {
+        var clear_btn_image = new Gtk.Image.from_icon_name ("edit-clear-all-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
+        clear_btn_image.get_style_context ().add_class ("sweep-animation");
+
+        var clear_btn_entry = new Gtk.Button () {
             tooltip_text = _("Clear all %s notifications").printf (name)
         };
+        clear_btn_entry.add (clear_btn_image);
```

Comment:
This should probably use the `child` property instead since `add ()` won't exist for Gtk.Button in GTK4